src/Conda.jl

```diff
@@ -189,6 +189,21 @@ _quiet() = get(ENV, "CI", "false") == "true" ? `-q` : ``
 "Install miniconda if it hasn't been installed yet; _install_conda(true) installs Conda even if it has already been installed."
 function _install_conda(env::Environment, force::Bool=false)
     if force || !isfile(Conda.conda)
+        if Sys.iswindows() && (!isascii(PREFIX) || ' ' ∈ PREFIX)
+            error("""Conda.jl cannot be installed to its default location $(PREFIX) as
+Miniconda does not support the installation to a directory with a non-ASCII
+character or a space on Windows. The work-around is to install Miniconda to a
+user-writable directory by setting the CONDA_JL_HOME environment
+variable before installing Conda.jl. For example:
+
+ENV["CONDA_JL_HOME"] = "C:\\\\Conda-julia\\\\3"
```

Comment:
Probably clearer to use `raw"C:\\Conda-Julia\3"`

Reply:
(`\\\\` gives a double backslash; not sure why that would be needed?)

Reply:
The idea would be that the user would just need to copy and paste the command. The error message would either print-out a double backslash or use `raw"C:\Conda-Julia\3"` (I use the latter in the updated PR).